### PR TITLE
Updated readmes, changelogs, and poms for new version

### PR DESCRIPTION
**Pull request by rickle-msft**

### CODE REVIEW

File: eng/jacoco-test-coverage/pom.xml [hunk 1]
```diff
@@ -234,27 +234,27 @@
...
       <groupId>com.azure</groupId>
       <artifactId>azure-storage-blob-batch</artifactId>
-      <version>12.8.0-beta.2</version> <!-- {x-version-update;com.azure:azure-storage-blob-batch;current} -->
+      <version>12.8.0</version> <!-- {x-version-update;com.azure:azure-storage-blob-batch;current} -->
     </dependency>
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-storage-blob-changefeed</artifactId>
-      <version>12.0.0-beta.5</version> <!-- {x-version-update;com.azure:azure-storage-blob-changefeed;current} -->
+      <version>12.0.0-beta.6</version> <!-- {x-version-update;com.azure:azure-storage-blob-changefeed;current} -->
```
Comment (gapra-msft) [inline eng/jacoco-test-coverage/pom.xml:252]: I think changefeed should be beta.5 still.

File: sdk/storage/azure-storage-blob-changefeed/CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,9 @@
 # Release History
 
-## 12.0.0-beta.5 (Unreleased)
+## 12.0.0-beta.5 (2010-01-14)
```
Comment (gapra-msft) [inline sdk/storage/azure-storage-blob-changefeed/CHANGELOG.md:3]: lol 2010?

File: sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/implementation/StorageImplUtils.java [hunk 3]
```diff
@@ -52,7 +52,7 @@
     private static final String STRING_TO_SIGN_LOG_WARNING_MESSAGE = "Please remember to disable '{}' before going "
         + "to production as this string can potentially contain PII.";
     private static final String STORAGE_EXCEPTION_LOG_STRING_TO_SIGN_MESSAGE = String.format(
-        "If you are using a StorageSharedKeyCredential, and the server returned an "
+        "If you are using a StorageSharedKeyCredential, and the serverv returned an "
```
Comment (gapra-msft) [inline sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/implementation/StorageImplUtils.java:55]: revert

**Review by alzimmermsft — APPROVED** [review]

**Pull request merged**